Makefile

```diff
@@ -33,32 +25,19 @@ checkgenerate: generate  ## Checks if `make generate` produces a diff.
 
 .PHONY: clean
 clean:  ## Delete intermediate build artifacts
-	@# -X only removes untracked files, -d recurses into directories, -f actually removes files/dirs
-	git clean -Xdf
```

Comment:
This was removing the `.idea/` directory used to work on this project too. By switching to Gradle (and ensuring all created files are created in `build/`), we can just use the normal clean step.